flow-editor/src/visual-node-editor/instance-view/InstanceView.tsx

```diff
@@ -168,7 +168,11 @@ export interface InstanceViewProps {
   onChangeStyle: (instance: EditorNodeInstance, style: NodeStyle) => void;
   onGroupSelected: () => void;
 
-  onPinMouseDown: (ins: EditorNodeInstance, pinId: string, type: PinType) => void;
+  onPinMouseDown: (
+    ins: EditorNodeInstance,
+    pinId: string,
+    type: PinType
+  ) => void;
```

Comment:
many formatting changes, please avoid them as it makes it hard to review + litters history